Products/ZenUI3/browser/resources/css/zen-cse.css

```diff
@@ -384,6 +384,14 @@ BYE BYE BORDERS
         padding: 2px 0 9px;
     }
 
+    /*ZEN-30098*/
+    .z-cse.z-cse-dark .x-surface text {
```

Comment:
All CSS definitions related to dark theme should go below the "dark theme" comment.

Reply:
Changed, please verify.